lib/portlayer/event/manager.go

```diff
@@ -72,7 +72,7 @@ func NewEventManager(collectors ...collector.Collector) *Manager {
 			log.Debugf("Found %d subscribers to id: %d - %s: %s", len(subs), e.EventID(), e.Topic(), e.Message())
 
 			for sub, s := range subs {
-				log.Debugf("Event manager calling back to %s", sub)
+				log.Debugf("Event manager calling back to %s for %d - %s: %s", sub, e.EventID(), e.Topic(), e.Message())
```

Comment:
Do we need the message & topic on this line since we have it on the line 72?  Just add eventID and then we are able to associate?

Reply:
Yes, otherwise we see lots of lines saying "Event manager calling back to exec" around the same area of the log file when we do things concurrently and no way to understand which one we missed etc.

Reply:
I was getting ready to say the event id allows us to tie it back...which I thought was the case, but since we are generating events w/o an id I understand why we'd need the dupe info...